hmda-analytics/src/main/scala/hmda/analytics/HmdaAnalyticsApp.scala

```diff
@@ -58,14 +58,14 @@ object HmdaAnalyticsApp
 
   val kafkaConfig = system.settings.config.getConfig("akka.kafka.consumer")
   val config = ConfigFactory.load()
-
+  val bankFilter = config.getConfig("filter")
+  val bankFilterList = bankFilter.getString("bank-filter-list").toUpperCase.split(",")
```

Comment:
Can we move these two lines in `processData` method? That way the `bankFilter` and `bankFilterList` will be read each time without requiring a restart of the pod when we make a change to the environment variable. For this to happen you'll need `config=ConfigFactory.load` inside `processData` as well

Reply:
we attempted this in hmda-regulator and the config still was not reloaded. Resulting in us having to scale down and restart the application regardless.